runtime/metrics_namespace.go

```diff
@@ -20,63 +20,49 @@
 
 package zanzibar
 
-import (
-	"net/http"
-
-	"github.com/uber/tchannel-go"
-)
+import "net/http"
 
 const (
-	inboundCallsRecvd        = "inbound.calls.recvd"
-	inboundCallsLatency      = "inbound.calls.latency"
-	inboundCallsSuccess      = "inbound.calls.success"
-	inboundCallsAppErrors    = "inbound.calls.app-errors"
-	inboundCallsSystemErrors = "inbound.calls.system-errors"
-	inboundCallsErrors       = "inbound.calls.errors"
-	inboundCallsStatus       = "inbound.calls.status"
+	endpointRequest      = "endpoint.request"
+	endpointSuccess      = "endpoint.success"
+	endpointStatus       = "endpoint.status"
+	endpointErrors       = "endpoint.errors"
+	endpointAppErrors    = "endpoint.app-errors"
+	endpointSystemErrors = "endpoint.system-errors"
+	endpointLatency      = "endpoint.latency"
 
-	// InboundCallsPanic is endpoint level panic counter
-	InboundCallsPanic = "inbound.calls.panic"
-
-	// TChannel docs say it emits 'outbound.calls.sent':
-	// http://tchannel.readthedocs.io/en/latest/metrics/#call-metrics
-	// but uber/tchannel-go emits 'outbound.calls.send':
-	// https://github.com/uber/tchannel-go/blob/3abb4c025c1663b383452339a22d918cf9d0be0b/outbound.go#L196
-	outboundCallsSend = "outbound.calls.send"
+	// EndpointPanics is endpoint level panic counter
+	EndpointPanics = "endpoint.panic"
 
-	outboundCallsSent         = "outbound.calls.sent"
-	outboundCallsLatency      = "outbound.calls.latency"
-	outboundCallsSuccess      = "outbound.calls.success"
-	outboundCallsAppErrors    = "outbound.calls.app-errors"
-	outboundCallsSystemErrors = "outbound.calls.system-errors"
-	outboundCallsErrors       = "outbound.calls.errors"
-	outboundCallsStatus       = "outbound.calls.status"
+	clientRequest   = "client.request"
+	clientSuccess   = "client.success"
+	clientStatus    = "client.status"
+	clientErrors    = "client.errors"
+	clientAppErrors = "client.app-errors"
```

Comment:
Shouldn't there be a `clientSystemErrors = "client.system-errors"`?

Reply:
fixed